src/app/pages/MarginTradePage/components/AddToMarginDialog/index.tsx

```diff
@@ -43,27 +42,33 @@ export const AddToMarginDialog: React.FC<IAddToMarginDialogProps> = ({
   onCloseModal,
 }) => {
   const canInteract = useCanInteract();
+  const {
+    id,
+    trade,
+    loanToken: { id: loanTokenId },
+    collateralToken: { id: collateralTokenId },
+  } = item;
+  const entryLeverage = trade?.[0].entryLeverage || '1';
+  const positionSizeValue = trade?.[0].positionSize || '0';
```

Comment:
Instead of string literals, pls create constants (`DEFAULT_ENTRY_LEVERAGE` and `DEFAULT_POSITION_SIZE`) in `MarginTradePage/types.ts` (because the same code is used in several places) and use them instead.